dashboard/test/ui/features/star_labs/farmer.feature

```diff
@@ -1,37 +1,34 @@
 Feature: Playing the Farmer Game
 
 Background:
-  Given I am on "http://studio.code.org/courses/20-hour/units/1/lessons/9/levels/1?noautoplay=true"
+  Given I am on "http://studio.code.org/courses/allthethingscourse/units/1/lessons/6/levels/1?noautoplay=true"
   And I wait for the lab page to fully load
   And I dismiss the login reminder
-  And element ".instructions-markdown p" has text "Hi, I'm a farmer. I need your help to flatten the field on my farm so it's ready for planting. Move me to the pile of dirt and use the \"remove\" block to remove it."
+  And element ".instructions-markdown p" has escaped text "Wow, look at that! I don't know how many shovelfuls of dirt this hole needs.\nCan you write a program that keeps using the fill block until the ground is even?  "
   Then element "#runButton" is visible
   And element "#resetButton" is hidden
 
 Scenario: Loading the first level
-  Then there's an image "video_thumbnails/farmer_intro"
   Then there's an image "farmer/small_static_avatar"
   Then I see "#pegman"
-  Then there's 1 dirt at (4, 4)
```

Comment:
The new level doesn't have a video and it's harder to check for a certain amount of dirt (it's a random amount at the one dirt tile), so I removed these. Videos thumbnails loading on levels is well covered in our test suite, so I don't think there's a loss of coverage there. Let me know if removing the check for dirt feels like too much of a loss of coverage and I can add a new level to allthethings.